var/www/cgi-bin/elucidator-restart.sh

```diff
@@ -36,7 +33,7 @@ function GetResponse {
     <section>
        <h1>Elucidator Restart Log</h1>
        <pre>"
-cat $RESTARTLOG
+ssh $ELUCIDATOR_HOST viewlog
```

Comment:
Did you consider `ssh -i /path/to/id_rsa ...` instead of relying on the key placement in ~/.ssh? I suppose both are similarly awkward, but mounting apache's .ssh directory seems a little more convoluted.